frontend/taipy-gui/src/components/Taipy/Selector.tsx

```diff
@@ -124,7 +153,7 @@ const getStyles = (id: string, ids: readonly string[], theme: Theme) => ({
 });
 
 const getOptionLabel = (option: LovItem) => (typeof option.item === "string" ? option.item : option.item?.text) || "";
-const getOptionKey = (option: LovItem) => "" + option.id;
+const getOptionKey = (option: LovItem) => `${option.id}`;
```

Comment:
Why this construct?
If id is a number... `toString()` will do the job right?

Reply:
Id is usually a string but could be a number...
This construct always returns a string whatever the value